src/CodeRepositoryProviders/FetchCriteria.php

```diff
@@ -6,7 +6,7 @@
 
 class FetchCriteria
 {
-    public function __construct(public readonly string $organizationName, public readonly string $providerName)
+    public function __construct(public readonly string $organizationName, public readonly string $providerName, public $accessKey)
```

Comment:
Jeżeli damy readonly nie będziem można potem zmienić tego accessKey i łatwiej nie zrobić błędu
```suggestion
    public function __construct(public readonly string $organizationName, public readonly string $providerName, public readonly string $accessKey)
```

https://php.watch/versions/8.1/readonly